ozaria/site/components/play/PageIntroLevel/index.vue

```diff
@@ -129,23 +124,40 @@
           this.currentIndex++
           if (this.currentIndex < this.introContent.length) { // increment current content
             this.currentContent = this.introContent[this.currentIndex]
+            this.setCurrentContentId(this.currentContent)
           } else { // whole intro content completed
             await this.setIntroLevelComplete()
```

Comment:
Is it possible for these async methods to fail?

Reply:
Yeah, but they are handling their own errors and will not return Promise.reject